share/static/html/home.html

```diff
@@ -70,12 +71,23 @@
   >
     {{notify_message}}
   </div>
+  {% if notify_url_to_image %}
+    <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label" style="width: 700px">
+        <input class="mdl-textfield__input" type="text" id="notify-url" value="{{notify_url_to_image}}" readonly>
+        <label class="mdl-textfield__label" for="notify-url">Your link to restore state</label>
+    </div>
+    <button class="mdl-button mdl-js-button mdl-button--icon notify-url--button notify-url__button" data-clipboard-target="#notify-url" title="Copy to clipboard">
+      <i class="material-icons">content_copy</i>
+    </button>
+  {% endif %}
 </div>
 
 {% endblock %}
 
 {% block script %}
 <script type="text/javascript">
-  require(["home"]);
+  require(["home", "clipboard"], function (home, Clipboard) {
+    new Clipboard('.notify-url__button');
+  });
```

Comment:
cool stuff! can you add the same copy-to-clipboard handlers to URLs of repo examples at the launch page?